src/api/procedures/removeCorporateAction.ts

```diff
@@ -42,30 +42,20 @@ const assertCaIsRemovable = async (
   context: Context,
   corporateAction: CorporateActionBase | BigNumber
 ): Promise<void> => {
-  const isBn = corporateAction instanceof BigNumber;
```

Comment:
(nit) I prefer this way since its checked twice. I think the var could be named `isId` also.

Reply:
Actually, I had to forcefully remove this because it was failing the test cases. Somehow in the `if` it was not automatically deducing the `corporateAction` can be a `BigNumber`

Reply:
Weird, but if it doesn't work the other way OK to leave